src/main/webapp/index.html

```diff
@@ -1,22 +1,23 @@
 <!DOCTYPE html>
-<html>
+<html lang='en'>
   <head>
     <meta charset="UTF-8">
     <title>Student Recommendations</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <script src="/scripts/mapscript.js"></script>
-    <script type='text/javascript' src='/scripts/config.js'></script>
+    <script src='/scripts/config.js'></script>
```

Comment:
nit
```suggestion
    <script src="/scripts/config.js"></script>
```
should we use double quote for all strings? just to make it consistent

Reply:
Sure, thanks!